### PR TITLE
Remove volume multiplier from spraycan

### LINKED ISSUE #17663: Spray cans completely insane now
What did this?

50 units of ethanol + 10 units of fuel for a single spray to the face?

Hardsuits+Masks don't even completely stop an application (4-6 units still go through)?

What?

### PR DESCRIPTION
**Pull request by optimumtact**
This was causing unintentionally massive doses of ethanol and welding fuel

Fixes #17663

### CODE REVIEW

**Comment by kevinz000** [comment]
Would rather just remove this so spraycans aren't a makeshift weapon.

File: code/game/objects/items/crayons.dm [hunk 1]
```diff
@@ -427,7 +427,7 @@
 				if(C.reagents)
 					C.reagents.add_reagent("welding_fuel", 5)
 					C.reagents.add_reagent("ethanol", 5)
-					C.reagents.reaction(C, VAPOR, 10)
+					C.reagents.reaction(C, VAPOR)
```
Comment (phil235) [inline code/game/objects/items/crayons.dm]: C.reagents.reaction(C) ...........
This is making the reagents inside you react with yourself.........

File: code/game/objects/items/crayons.dm [hunk 2]
```diff
@@ -427,7 +427,7 @@
 				if(C.reagents)
 					C.reagents.add_reagent("welding_fuel", 5)
 					C.reagents.add_reagent("ethanol", 5)
```
Comment (phil235) [inline code/game/objects/items/crayons.dm]: yes, let's just ignore any sort of protectiong the mob could have....

**Comment by phil235** [comment]
This doesn't fix the whole thing. The code is still terrible.

**Comment by optimumtact** [comment]
I'll have a gander tomorrow

**Comment by coiax** [comment]
Yeah, it should check for eye protection before injecting the reagents.

@kevinz000 this was during adding the spraycan to service/standard borg, needed the spraycan to do damage so they couldn't get a harm-free stun.

File: code/game/objects/items/crayons.dm (hunk 2 above)
Comment (PKPenguin321) [inline code/game/objects/items/crayons.dm]: Check pocket sand or paper airplanes throw_impact, both have checks for eye protection that you can probably just copy and paste

**Comment by optimumtact** [comment]
@phil235 are we doing this right now or are we still misunderstanding chem code?

**Comment by phil235** [comment]
The code still has a huge flaw with the reation() call. What it currently does add ethanol/fuel into you (which isn't very realistic, you're not drinking from the spraycan but at this point this isn't even the worst problem) then what happens is exaclty as if we copied all the reagents inside you into a spray and sprayed you with it which , if you don't have full permeability protection, will add a percentage of those reagents into you. 

Or to make it short: `C.reagents.reaction(C, VAPOR, 1 / divisor)` is multiplying all the reagents inside you by roughly 1.5 every time you're sprayed with the spraycan.

**Comment by optimumtact** [comment]
how do we apply the reagents as if they were sprayed on to the user then? do we create a custom reagent holder, apply a vapor reaction to it  and then apply the result to the user in a volume that matches their protection level?

**Comment by phil235** [comment]
Well if you don't want to fully transform spraycans into actual sprays, you need to create a reagents holder, add the fuel and ethanol to it, then call holderreagents.react(C, VAPOR), then you qdel(holderreagents). It's a bit hacky but you can't really do otherwise when you try to apply reagents that come out of nowhere.

**Comment by optimumtact** [comment]
fixed by coaix in #17815

**Pull request closed without merging**